neutron/chain.json

```diff
@@ -248,20 +253,58 @@
           "type": "go",
           "version": "v8.2.1"
         }
+      },
+      {
+        "name": "v4.2.1",
```

Comment:
Seems like there should be a single version object for all v4.2 versions

Reply:
But it should be applied only at specific height, how can i specify it in a single version object?

Reply:
the block height number to use should be when v4.2.0 started. The versions of components (language, sdk, ibc, cosmwasm, etc.) and the binaries should reflect the latest well-running/recommended version. Since v4.2.0, v4.2.1, and v4.2.2 are all compatible versions, it should be inconsequential to use the earliest version or the earlier version of components.